app/controllers/hyrax/downloads_controller.rb

```diff
@@ -41,6 +41,7 @@ def derivative_download_options
       # Hydra::Ability#download_permissions can't be used in this case because it assumes
       # that files are in a LDP basic container, and thus, included in the asset's uri.
       def authorize_download!
+        return if current_ability.admin?
```

Comment:
Yeah I agree but tried all kinds of ways to do that and my CanCan(Can) knowledge isn't enough to see what to do.  The thing is, the admin user already has [declared ability on ::FileSet in Ability](https://github.com/samvera/hyrax/blob/master/app/models/concerns/hyrax/ability.rb#L207), and presumably that's what gets the admin user access to the parent file_set in the file_set controller context.  It should just work without the shortcut but I can try a different shortcut in Ability if someone can tell me what to do.